### DISCUSSION TITLE
Support from the Sovereign Tech Fund in 2024 and 2025

### DISCUSSION THREAD

**Discussion by marcphilipp** [junit-team/junit-framework#4014] [Announcements]
I am happy to announce that the [Sovereign Tech Fund](https://www.sovereigntechfund.de/tech/junit) will invest in JUnit in 2024 and 2025. That means I will get to work full-time on JUnit for about a year! :tada: 

The _Sovereign Tech Fund_ is a German government program that aims to strengthen the open source ecosystem by investing in foundational technologies that enable the creation of other software. We are very happy that JUnit is considered one of those!

The work is comprised of a set of predefined milestones that have been agreed upon with the Sovereign Tech Fund. The goal is to improve the long-term sustainability of the project as well as downstream projects.

To clarify, JUnit will remain an independent project, and the investment does not mean that the Sovereign Tech Fund will be influencing JUnit's roadmap apart from the milestones they are commissioning.

You can follow along via this [GitHub project](https://github.com/orgs/junit-team/projects/4/views/1).

**Comment by jfelrod1960** [comment]
Happy for you and the JUnit effort!  Best wishes! :)

**Comment by mpkorstanje** [comment]
Congratulations! 🎉

**Comment by khmarbaise** [comment]
Congratulations! 🚀 💯 👍

**Comment by vdmitrienko** [comment]
Great news! 
Congratulations! 🎉

**Comment by YongGoose** [comment]
Congratulations on this exciting news! 🎉

**Comment by marcphilipp** [comment]
I have started a series of blog posts on my work on JUnit supported by the STF: https://marcphilipp.de/blog/2025/01/19/being-a-full-time-open-source-maintainer-supported-by-the-sovereign-tech-fund/

**Reply by YongGoose** [reply]
That sounds like an exciting initiative! 
Looking forward to reading your blog posts and learning more about your work on JUnit with the STF. 
Keep up the great work 💪🏻

**Comment by marcphilipp** [comment]
In case you’re interested, I‘ve now also written a wrap-up post: https://marcphilipp.de/blog/2026/08/02/reflecting-on-stf-support-and-looking-ahead/
